changelog.txt

```diff
@@ -1,3 +1,6 @@
+Every operand of a String concatenation can be "null", unless it is a String literal,
+primitive or compile-time constant.
```

Comment:
This is a minor bug fix, which does not need to go in the changelog.
Any new or changed command-line argument needs to be in the changelog, however.  Could you please add a note about the command-line argument?